src/datagrid/DataGridValues.tsx

```diff
@@ -120,35 +133,39 @@ export const filterFunction = (
     });
 };
 
+/**
+ * Data for Sorting
+ */
+
 // Custom sorting function for number and string type
 export const sortFunction = (rows: DataGridRow[], sortOrder: SortOrder, columnName: string): Promise<DataGridRow[]> => {
     return new Promise((resolve, reject) => {
         rows.sort(function(first: DataGridRow, second: DataGridRow): number {
```

Comment:
let's prefer arrow function for anonymous functions

Reply:
done